src/useNotification.ts

```diff
@@ -0,0 +1,30 @@
+import { useEffect, useState } from 'react';
+import * as thisModule from './useNotification';
```

Comment:
Maybe we should not import this module in itself?

Reply:
As above